tools/export/uvision5.py

```diff
@@ -83,11 +83,14 @@ def generate(self):
                                                    + self.flags['c_flags']
                                                    + self.flags['cxx_flags']))
         # not compatible with c99 flag set in the template
-        project_data['misc']['c_flags'].remove("--c99")
+        try:project_data['misc']['c_flags'].remove("--c99")
+        except ValueError: pass
```

Comment:
Given how many times that we do this, we should have a utility function that does it.